evergreen.yml

```diff
@@ -1605,6 +1617,7 @@ tasks:
       - name: release
     commands:
       - func: "update download center feed"
+      - func: "trace artifacts"
```

Comment:
Let's put this before "update download center feed" so that if the download center feed update fails, we still traced the released artifacts, and we can fix the feed manually if necessary (it happened before unfortunately)